### DISCUSSION TITLE
客户端获取

### DISCUSSION THREAD

**Discussion by qwesada1233** [Melledy/Grasscutter#1795] [Q&A]
请问在哪里可以下载相应的客户端文件

**Comment by qwqlemon2333** [comment]
额。。。。普通的客户端就行了呀

**Reply by qwesada1233** [reply]
> 额。。。。普通的客户端就行了呀

直接使用官服的客户端吗？

**Comment by 2y8e9h22** [comment]
原神官网

**Reply by qwesada1233** [reply]
> 原神官网

如果我私服是搭建的3.1或者3.2版本的咋搞呢（我看服务器好像有3.2版本的了）

**Reply by qinfyy** [reply]
没有那种东西

**Comment by zhnyyds** [comment]
怎么下载私服呀

**Comment by xx065141** [comment]
ios怎么整

**Reply by Musicminion** [reply]
参考 https://github.com/Grasscutters/Grasscutter/discussions/2050
或者 https://blog.csdn.net/weixin_51394621/article/details/128894111
